importer.test.js

```diff
@@ -0,0 +1,59 @@
+jest.mock('find-parent-dir');
+jest.mock('./fs-util');
+
+var importer = require('./index');
```

Comment:
this file should be called `index.test.js`

Reply:
ok, I'll rename it